### ISSUE TITLE
useA11yEnabled cleanup callback is throwing an error on development

### ISSUE THREAD

**Issue by leobastiani** [ArturKalach/react-native-a11y#32]
I encountered an error during development, and I'm unsure if this issue is occurring in production (although it's likely not).

![image](https://github.com/ArturKalach/react-native-a11y/assets/8663061/1cd853cd-a06b-4da1-abcc-b45ad5b58315)

I made a research and I found that this is the [line](https://github.com/facebook/react-native/blob/0.68-stable/Libraries/vendor/emitter/_EmitterSubscription.js#L58) of the error and `this` is not assigned because [here](https://github.com/ArturKalach/react-native-a11y/blob/b16db768669b3d9351273b65455138b3fafbf93b/src/hooks/useA11yEnabled/useA11yEnabled.ts#L13) we are using

```ts
return listener.remove;
```

and we lose `this` pointer

**the best** alternative for me is to use

```ts
return () => listener.remove();
```

but we could also use

```ts
return listener.remove.bind(listener);
```

**Comment by ArturKalach** [comment]
Hello @leobastiani,
Thank you a lot for a help and creating issue.

This has been fixed with version 0.3.2.

I won't close this issue until I fix it in "0.3.0-android.2" which is used for supporting old  version of RN

**Comment by ArturKalach** [comment]
- Old React Native versions support approach has been changed with `0.4.2` version.
It means that this issue has been fixed for "all" versions.
